labgrid/driver/sshdriver.py

```diff
@@ -111,12 +110,19 @@ def _start_own_master_once(self, timeout):
             ]
 
         env = os.environ.copy()
+        pass_file = ''
         if self.networkservice.password:
-            env['SSHPASS'] = self.networkservice.password
+            fd, pass_file = tempfile.mkstemp()
+            os.fchmod(fd, stat.S_IRWXU)
+            env = {'SSH_ASKPASS': pass_file, 'DISPLAY':'0:0'}
+            with open(fd, 'w') as f:
+                f.write("echo " + self.networkservice.password)
```

Comment:
This needs `#!/bin/sh\n`.

Reply:
Hmm, this could also be used to do something like `echo $LG_SSH_PASSWORD`, right?